controllers/authController.js

```diff
@@ -50,6 +53,7 @@ exports.register = async (req, res) => {
       })
     }
 
+    req.body.password = bcrypt.hashSync(req.body.password, salt)
```

Comment:
here we should use the technique 2 to hash this w/o salt, using only the salt rounds.